File: practice/programiz/roots_of_quadratic_equation.c
#include <stdio.h>

/**
 * @brief in this program find the roots of quadratic equation
 */
int main(void)
{
    int a, b, c;
    printf("enter a value of a:\n");
    scanf("%i", &a);
    printf("enter a value of b:\n");
    scanf("%i", &b);
    printf("enter a value of c:\n");
    scanf("%i", &c);
    int Discriminant = (b * b - 4 * a * c);
    if (Discriminant > 0)
    {
        printf("two different real roots\n");
    }
    else if (Discriminant == 0)
    {
        printf("two real some roots\n");
    }
    else if (Discriminant < 0)
    {
        printf("not real roots\n");
    }

    return 0;
}

File: practice/java/RootsOfQuadraticEquation.java

import java.util.Scanner;

/**
 * @brief this program find the roots of quadratic equation
 * RootsOfQuadraticEquation
 */
class RootsOfQuadraticEquation {

    public static void main(String[] args) {
        int a;
        int b;
        int c;
        Scanner scanner = new Scanner(System.in);
        System.out.println("enter a value of a:");
        a = scanner.nextInt();
        System.out.println("enter a value of b:");
        b = scanner.nextInt();
        System.out.println("enter a value of c:");
        c = scanner.nextInt();
        scanner.close();
        int discriminant = (b * b - 4 * a * c);
        if (discriminant > 0) {
            System.out.println("two different reals roots");
        } else if (discriminant == 0) {
            System.out.println("two same real roots");

        } else if (discriminant < 0) {
            System.out.println("not real roots");
        }

    }
}
